tests/Unit/Extensions/ImagesExtensionTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Xefi\Faker\Images\Tests\Unit\Extensions;
+
+use Intervention\Image\Image;
+use Random\Randomizer;
+use Xefi\Faker\Container\Container;
+use Xefi\Faker\Images\Exceptions\NoDriverException;
+use Xefi\Faker\Images\Tests\Unit\TestCase;
+
+final class ImagesExtensionTest extends TestCase
+{
+    public function testImage(): void
+    {
+        $faker = new Container(false);
+
+        if (!extension_loaded('gd') && !extension_loaded('imagick')) {
+            $this->expectException(NoDriverException::class);
+        }
+
+        $image = $faker->unique()->image();
+
+        if (extension_loaded('gd') || extension_loaded('imagick')) {
+            $this->assertInstanceOf(Image::class, $image);
+            $this->assertEquals(300, $image->width());
+            $this->assertEquals(200, $image->height());
+        }
+    }
+
+    public function testImageWithCustomParameters(): void
+    {
+        $faker = new Container(false);
+        $randomizer = new Randomizer();
+
+        for ($i = 0; $i < 10; $i++) {
+            $width = $randomizer->getInt(1, 2000);
+            $height = $randomizer->getInt(1, 2000);
+
+            if (!extension_loaded('gd') && !extension_loaded('imagick')) {
+                $this->expectException(NoDriverException::class);
+            }
+
+            $image = $faker->unique()->image($width, $height);
+
+            if (extension_loaded('gd') || extension_loaded('imagick')) {
+                $this->assertInstanceOf(Image::class, $image);
+                $this->assertEquals($width, $image->width());
+                $this->assertEquals($height, $image->height());
+            }
+        }
```

Comment:
And for the rest, just skip if there are no extensions, this feels weird, please have a look also if you can maybe skip with an annotation ?

Reply:
![image](https://github.com/user-attachments/assets/a04408cd-d863-4fc0-91a9-02fc7c67687c)
maybe

Reply:
I used it now so it clearer :)